### DISCUSSION TITLE
Vector Tiles support?

### DISCUSSION THREAD

**Discussion by FrancoisDuquesne** [developmentseed/titiler#945] [Q&A] [answered]
Hey!

Do you plan on supporting vector tiles like .mvt, .mbtiles, .pbf files? 
Or is there a way to serve them currently ?

Thx,

**Comment by vincentsarago** [comment, marked as answer]
Hi @FrancoisDuquesne 

TiTiler focuses only on Raster dataset.

For `mvt` or `pbf` you do not need a complex solution like titiler because it's just a matter of serving files from a storage (e.g AWS S3 + cloudfront will be more than enough).

For `mbtiles` you'll need to have something between your client application and you `mbtile` file. There are multiple solution that exist for this.... but I will encourage you to look at `PMTILES` which seems to be the new `default` format for storing files in one archive

**Reply by FrancoisDuquesne** [reply]
Ok great, thx for the answer!
